ckanext/basiccharts/plugin.py

```diff
@@ -225,7 +237,7 @@ def convert_to_string(value):
 
 def validate_fields(key, converted_data, errors, context):
     try:
-        resource = {'id': converted_data['resource_id',]}
+        resource = {'id': converted_data['resource_id', ]}
```

Comment:
I don't think this comma and whitespace are supposed to be there.
I think this is a dict lookup? unless is syntax thing for pylons...